doc/api/configuration.md

```diff
@@ -234,6 +234,22 @@ value enables the ones from previous values plus new ones:
 - `2`: All feedback
   - Footer to distinguish pandas/polars data frames from JupySQL's result sets
 
+## `lazy_execution`
+
+Default: `False`
```

Comment:
I'm guessing this only applies to spark? if so, it'd be better to mention it and to rename this to `spark_lazy_execution`

Reply:
I'd figured you could extend to duckdb RE: https://github.com/ploomber/jupysql/issues/53, if not interested in carrying that forward I'll rename back to spark only?

Reply:
makes sense. let's keep `lazy_execution` then, just add a version added note explaining that this only works with spark:

~~~
```{versionadded} 0.10.7
This option only works when connecting to Spark
```
~~~

```

Reply:
Adjusted https://github.com/ploomber/jupysql/pull/968/files/36aaa1003f50c75188b6f0cf169d3a7157d53645..db024fb2149b3b050f6ea0507c4cc36a3da77cfc#diff-7c559917b46c06e53b27fa5152bf29cb1db41f04c53f857ef5331f7a003204ffR239-R241